src/_scss/pages/search/searchSidebar.scss

```diff
@@ -8,40 +8,77 @@
     box-shadow: $container-shadow;
     border-radius: rem(5);
     margin-top: rem(32);
+
     .sidebar-header {
         display: none;
-        @include media($tablet-screen) {
-            @include display(flex);
-            @include align-items(center);
-            @include flex-flow(row);
+        @include media($small-screen) {
+            display: inline-table;
+            @include align-items(left);
             @include justify-content(flex-start);
             position: relative;
             border-bottom: rem(1) solid $color-gray-lighter;
             height: rem(70);
+            width: 100%;
             margin: 0;
             padding: 0 0 0 rem(15);
```

Comment:
Should change this be padding: 0 rem(16);
We need padding right and left now that the text is long enough to hit the right edge, and we should change it from 15 to 16 bc of our new design specs.

![20230314FilterTextPaddingRight](https://user-images.githubusercontent.com/49243108/225136420-0e1f940b-b210-4011-9020-2d933af0cb87.png)